src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -238,7 +240,12 @@
 )
 
 
-async def initialize(server_url: str, resampler_config: ResamplerConfig) -> None:
+async def initialize(
+    server_url: str,
+    resampler_config: ResamplerConfig,
+    *,
+    api_power_request_timeout: timedelta = timedelta(seconds=5.0),
```

Comment:
Thinking about the future direction, maybe it would make sense to have some other object, like `MicrogridApiConfig` (or if this is actually a power manager specific thing, `PowerManagerConfig`). If we do this, adding more config related to either the microgrid API or the power manager can be done in a backwards compatible way (by adding more fields to the new config object) and without adding a lot of other keyword arguments to `initialize()`.